README.md

```diff
@@ -35,21 +35,23 @@ workaround:
 
 ## How to install *realpath_turbo*
 
-1. Download the *realpath_turbo* source code.
-
-2. Extract and compile the extension
-
-   ```sh
-   $ tar -xaf realpath_turbo*
-   $ cd realpath_turbo*
-   $ phpize
-   $ ./configure
-   $ make
-   # make install
-   ```
-
-3. Adjust your `php.ini` to load and configure *realpath_turbo* extension.
-
+```sh
+wget -qO- https://github.com/Whissi/realpath_turbo/archive/master.tar.gz | tar -xz
+cd realpath_turbo-master/
+
+phpize
+./configure
+make
+make test NO_INTERACTION=1; echo $?
+make "INSTALL=$(pwd)/build/shtool install -c --mode=0644" install
+
+# PHP 5.*
+#echo -e "; priority=20\nextension=realpath_turbo.so" > /etc/php5/mods-available/realpath_turbo.ini
+#php5enmod realpath_turbo
+echo -e "; priority=20\nextension=realpath_turbo.so" > /etc/php/7.0/mods-available/realpath_turbo.ini
+phpenmod realpath_turbo
+php -m | grep realpath_turbo

```

Comment:
Adding a hint how to verify that *realpath_turbo* is available is a good idea!
However that's not an easy task: On my platform for example I have separate CFGs for CLI and non-CLI usage...

Also: I fear that people maybe only do this check and forget about `realplath_turbo`. However without set `realpath_turbo.open_basedir` config directive the extension wouldn't do anything... so I think if we add a guide how to verify that the extension is working we should also make sure that people don't forget about the CFG and how they verify a valid configuration.